types/dompurify/index.d.ts

```diff
@@ -3,26 +3,37 @@
 // Definitions by: Dave Taylor <http://davetayls.me>, Samira Bazuzi <https://github.com/bazuzi>, FlowCrypt <https://github.com/FlowCrypt>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export as namespace DOMPurify;
+export = createDOMPurify
```

Comment:
It looks like the new form isn't consumable as a global.  Was that intentional?

Reply:
@amcasey can you explain the difference? I am not familiar with typings, I just write as I know and test it.

Reply:
I think of the `as namespace` version as being a way to make the declarations available in non-modules (e.g. webpages).  There's a more thorough explanation here: https://www.typescriptlang.org/docs/handbook/modules.html#umd-modules

Reply:
Should I rename DOMPurify interface to DOMPurifyI and export createDOMPurify as DOMPurify namespace?

Reply:
I figured out. What you think, maybe we should add this into the FAQ?